ui/lib/pki/addon/components/pki-configure-form.hbs

```diff
@@ -30,7 +30,7 @@
     </div>
   {{/unless}}
   {{#if (eq @config.actionType "import")}}
-    <PkiCaCertificateImport
```

Comment:
happy to revert if this change feels unnecessary/unhelpful

Reply:
Works for me!